modules/services/gpg-agent.nix

```diff
@@ -129,6 +129,22 @@ in
           configuration file.
         '';
       };
+
+      pinentryFlavor = mkOption {
+        type = types.nullOr (types.enum pkgs.pinentry.flavors);
+        example = "gnome3";
+        default = "gtk2";
+        description = ''
+          Which pinentry interface to use. If not null, the path to the
```

Comment:
I think the wording here needs to change for HM since this value actually goes inside the `gpg-agent.conf` and is not passed as a command line argument. This is almost the exact opposite of what the description says 🙂